src/newFunctionalities.js

```diff
@@ -0,0 +1,39 @@
+export const inputTask = document.getElementById('input-area');
+export const addButton = document.querySelector('.addButton');
+// eslint-disable-next-line import/no-mutable-exports
+export let tasksData = JSON.parse(localStorage.getItem('task')) || [];
+
+const taskRemover = (tasknumber) => {
+  tasksData = tasksData.filter((dailyTasks) => dailyTasks.index !== tasknumber);
+  localStorage.setItem('task', JSON.stringify(tasksData));
+  window.location.reload();
+};
+export function executeData(input) {
+  input.forEach((element, i) => {
+    element.index = i + 1;
```

Comment:
- [ ] Actually, there is nothing that works 👇🏻 

[Screencast from 02-24-2023 09:43:58 PM.webm](https://user-images.githubusercontent.com/79058364/221276368-00434e38-19fb-4fb4-88a8-89016202acb8.webm)

Kindly make sure:
1. A function for adding a new book is well implemented. ✔️   
2. A function for editing task descriptions is well implemented. ✔️   
3. A function for remove a book from the array is implemented. ✔️   
4.  Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. ✔️   
5. All changes to the To-Do List should be saved in local storage. ✔️